cl/opinion_page/views.py

```diff
@@ -79,6 +82,36 @@ def court_homepage(request, pk):
     return render(request, "court.html", render_dict)
 
 
+@group_required("tenn_work_uploaders")
+def court_publishpage(request, pk):
+    """Display uploader and intake publication of doc for Tenn. Workers Comp
+
+    :param request:
+    :param pk:
+    :return:
+    """
+
+    if pk not in ["tennworkcompcl", "tennworkcompapp"]:
+        raise Http404("Court pages only implemented for Tennessee so far.")
+    form = TennWorkersForm(pk=pk)
+    if request.method == "POST":
+        form = TennWorkersForm(request.POST, request.FILES, pk=pk)
+        if form.is_valid():
+            cluster_id = form.save()
+            goto = "https://www.courtlistener.com/opinion/%s/go" % cluster_id
+            messages.info(
+                request, "Document uploaded successfully.", extra_tags=goto
+            )
+            return HttpResponseRedirect(
+                reverse("court_publishpage", kwargs={"pk": pk})
+            )
+        else:
+            messages.info(request, "Form incomplete, please review the form.")
```

Comment:
I'd say something like, "Error submitting form, please review below", because "Incomplete" may not be right.

Reply:
√